### ISSUE TITLE
Docker image (intelanalytics/ipex-llm-xpu): Documentation stated I would need to disable iGPU to use A770. When will you fix this issue since disabling iGPU is problematic?

### ISSUE THREAD

**Issue by sungkim11** [intel-analytics/ipex-llm#10940]
Docker image (intelanalytics/ipex-llm-xpu): Documentation stated I would need to disable iGPU to use A770. When will you fix this issue since disabling iGPU is problematic?

**Comment by digitalscream** [comment]
What are you using it for? You might have some luck with setting this environment variable (after the `setvars.sh` and `source ipex-llm-init` lines):

export ONEAPI_DEVICE_SELECTOR=level_zero:0

Set it to `level_zero:0` or `level_zero:1`, see if one of them works. I used this strategy when I was experimenting with dual A770s, one on x4 PCIE and one on x16.

More documentation here:

https://intel.github.io/llvm-docs/EnvironmentVariables.html

**Comment by liu-shaojun** [comment]
Hi @sungkim11 are you following this document: https://ipex-llm.readthedocs.io/en/latest/doc/LLM/Quickstart/docker_windows_gpu.html# to install IPEX-LLM in Docker, we will try to reproduce from our side and try the method suggested by @digitalscream.

**Comment by sungkim11** [comment]
Yes, I have followed the document. I am referencing the last section where it suggest I disable iGPU. This is problematic for me.

**Comment by sungkim11** [comment]
This is problematic since my monitors are using iGPU, not A770. I would lose display when I disable iGPU.

**Comment by liu-shaojun** [comment]
This issue might be fixed by upgrading intel-extension-for-pytorch. However, the new ipex requires OneAPI 2024.1 and we haven't tested ipex-llm with OneAPI 2024.1 yet. If the test succeeds later on, we'll update the issue. Alternatively, you can try using A770 for graphical interface support.
